awscli/topics/config-vars.rst

```diff
@@ -462,11 +462,19 @@ One option for UNIX systems is the ``LC_ALL`` environment variable. Setting
 ``LC_ALL=en_US.UTF-8``, for instance, would give you a United States English
 locale which is compatible with unicode.
 
-To set encoding used for text files different from the locale, you can use
+To set the encoding that is used when reading from text files, you can use the
 ``AWS_CLI_FILE_ENCODING`` environment variable. For example, if you use Windows
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+To set the encoding used for the CLI's output, you can use the
+``AWS_CLI_OUTPUT_ENCODING`` environment variable. For example, if you use Windows
+with default encoding ``CP1252``, setting ``AWS_CLI_OUTPUT_ENCODING=UTF-8`` would
+make CLI ignore the locale encoding and format its output using ``UTF-8``.
+
+Recent versions of the AWS CLI v2 may no longer respect Python's environment
+variables for configuring encoding, such as ``PYTHONUTF8`` and ``PYTHONIOENCODING``.
```

Comment:
nonblocking: Maybe link to valid encodings in Python? https://docs.python.org/3/library/codecs.html#standard-encodings

Reply:
Added via https://github.com/aws/aws-cli/pull/9319/commits/fdeac1e8079c527e42d03ca54893272c31242f06